packages/utils/telemetry-utils/src/test/multiSinkLogger.spec.ts

```diff
@@ -57,4 +59,38 @@ describe("MultiSinkLogger", () => {
 		logger1.assertMatch([{ category: "generic", eventName: "test", test: true }]);
 		logger2.assertMatch([{ category: "generic", eventName: "test" }]);
 	});
+
+	it("MultiSink logger set the logLevel to min logLevel of all loggers", () => {
+		const logger1 = new MockLogger(LogLevel.error);
+		const logger2 = new MockLogger(LogLevel.error);
```

Comment:
Nit: this test would probably be a bit more thorough if we make one of these two `default` and check for that (so it tests that the MultiSinkLogger reports the correct minLogLevel right after instantiation; and the code below already exercises that for when a new logger is added).